tests/func/test_data_cloud.py

```diff
@@ -327,6 +327,16 @@ def _test_cloud(self):
             with open(cache, "r") as fd:
                 self.assertEqual(fd.read(), self.FOO_CONTENTS)
 
+            remote = self.cloud._cloud
```

Comment:
Let's not test this in this functional test. This is not cloud-specific and could be handled separately. For example as a unit test.